includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -23,22 +33,25 @@ public function checkAntiSpoof() {
 		$this->response->setCacheValidity( \WikiaResponse::CACHE_DISABLED );
 
 		if ( !$this->authenticateViaTheSchwartz() ) {
-			$this->response->setVal( 'allow', false );
+			$this->response->setVal( self::FIELD_ALLOW, false );
+
 			return;
 		}
 
 		$username = $this->getVal( 'username' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "username" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)